### PR TITLE
rpi: set UBOOT_MACHINE variable for building u-boot

### PR DESCRIPTION
**Pull request by yuwata**

### CODE REVIEW

**Comment by masami256** [comment]
@yuwata Does [raspberrypi3.md](https://github.com/miraclelinux/meta-emlinux/blob/warrior/doc/raspberrypi3.md)  need to update?

**Comment by yuwata** [comment]
Ah, not sure. I will check it.

**Comment by yuwata** [comment]
@masami256 Updated. PTAL.

File: doc/raspberrypi3.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ In this document, following directory structure is used.
 ```
 build/
 downloads/
-emlinux/repos/meta-debian 
+emlinux/repos/meta-debian
 emlinux/repos/meta-debian-extended
 emlinux/repos/meta-emlinux
 emlinux/repos/poky
```
Comment (masami256) [inline doc/raspberrypi3.md:16]: @yuwata In the "Directory structures" section,  you can remove u-boot directory.

**Comment by masami256** [comment]
Tested on rpi3 b+, it works fine for me.

**Comment by yuwata** [comment]
@masami256 Thank you for the review. Updated. PTAL.

**Comment by masami256** [comment]
LGTM!

**Pull request merged**